### PR TITLE
styler patch

### PR DESCRIPTION
**Pull request by dylanhmorris**

### CODE REVIEW

File: .github/workflows/pre-commit.yaml [hunk 1]
```diff
@@ -13,9 +13,9 @@ jobs:
...
       with:
         python-version: "3.12"
-    - uses: actions/cache@v3
+    - uses: actions/cache@v4
       with:
         path: |
-          ~/.cache/pre-commit
           ~/.cache/R/renv
-        key: pre-commit-3|${{ env.pythonLocation }}|${{ hashFiles('.pre-commit-config.yaml') }}
+        key: r-precommit-cache|${{ env.pythonLocation }}|${{ hashFiles('.pre-commit-config.yaml') }}
```
Comment (kaitejohnson) [inline .github/workflows/pre-commit.yaml:20]: So was the key thing here that we needed to point to an `r-precommit-cache`? Just trying to understand this at a high level.

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -35,7 +35,8 @@ repos:
     rev: v0.4.2
     hooks:
     -   id: style-files
-        args: [--style_pkg=styler, --style_fun=tidyverse_style]
+        args: [--style_pkg=styler, --style_fun=tidyverse_style,
+               --cache-root=styler-perm]
```
Comment (kaitejohnson) [inline .pre-commit-config.yaml:39]: Is it worth updating what is in here on the cdcent template repo?

**Review by kaitejohnson — APPROVED** [review]
LGTM, just some clarifying questions

**Comment by kaitejohnson** [comment]
@dylanhmorris I can merge this into #1, then I think I just need to fix the `pkgdown`

File: .pre-commit-config.yaml (hunk 2 above)
Comment (dylanhmorris) [reply .pre-commit-config.yaml:39]: Arguably but this isn't as critical as the use of `actions/cache`

**Comment by dylanhmorris** [comment]
Sounds good will let you merge @kaitejohnson

**Pull request merged**